pipeline_dp/accumulator.py

```diff
@@ -4,6 +4,7 @@
 from dataclasses import dataclass
 from functools import reduce
 import pipeline_dp
+from pipeline_dp.dp_computations import MeanVarParams
```

Comment:
By [styleguide](https://google.github.io/styleguide/pyguide.html#22-imports), only modules should be imported (with some standard classes exceptions).

Reply:
Done